frontend/src/js/components/common/forms/textinput.js

```diff
@@ -32,6 +32,7 @@ export const TextInput = ({
   required,
   type,
   validations = '',
+  numericValidations = null,
```

Comment:
```suggestion
  numericValidationProps = {},
```
as you spread them anyway in the rules